.gitignore

```diff
@@ -1,2 +1,3 @@
 /vendor
-/codeCoverage
\ No newline at end of file
+/codeCoverage
+.idea
```

Comment:
Is it traditional to include IDE specific project files?
I've just put mine in .git/info/exclude to try and keep this clean